requirements-base.txt

```diff
@@ -79,7 +79,7 @@ kombu==4.6.11
 grpcio==1.40.0
 
 # not directly used, but provides a speedup for redis
-hiredis==0.3.1
+hiredis==2.0.0
```

Comment:
Last time I did this https://github.com/getsentry/sentry/pull/25238, ingestion totally broke because

> strict unicode decode errors which is passed through with default true from redis-py ≥3.3.0 to hiredis-py ≥ 1.0.0

Reply:
I see. I remember you tried something. I wanted to get in the `redis-py 4` as they introduced the `Retry` configurations but the `redis-py-cluster` is not yet compatible with `redis-py 4` therefore I closed this PR.